### ISSUE TITLE
Bug - show just first item in subnav

### ISSUE THREAD

**Issue by wmdhosting** [engram-design/CPNav#116]
### Describe the bug

Last update fixed many things.  but now 
its show only first... item in subnav.. 
![Snag_15357a1b](https://user-images.githubusercontent.com/76176291/198220650-ed7e0781-07a7-4503-9798-41a0be059e6d.png)

### Steps to reproduce

and geting error when i click on  CP clear cache icon..  normaly opens as modal.. 
![Snag_1535e76b](https://user-images.githubusercontent.com/76176291/198220767-c1a2ed1b-4435-4bb8-aeac-a4f384547a29.png)

### Craft CMS version

4.3

### Plugin version

4.0.7

### Multi-site?

Y

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/cp-nav` requirement in `composer.json` to:

```
"require": {
  "verbb/cp-nav": "dev-craft-4 as 4.0.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by wmdhosting** [comment]
please can you tag new version. 
for some reason. whatever i tryed .. cant get dev fix..

**Comment by engram-design** [comment]
What's the issue with using the `dev-craft-4` version? Are you getting an error when you try to install it?

Regardless should be fixed in [4.0.8](https://github.com/verbb/cp-nav/releases/tag/4.0.8)

**Comment by wmdhosting** [comment]
Just info..     its ok. now for  plugins.. 

But not for manual items.. 
![Snag_11567aea](https://user-images.githubusercontent.com/76176291/200871500-74812490-556f-41e3-9a74-f50bf083433b.png)

**Comment by engram-design** [comment]
Properly fixed in [4.0.9](https://github.com/verbb/cp-nav/releases/tag/4.0.9). Please note that you'll need to move something in or out of your subnav items to apply this fix. It had to do with the subnav items not properly recording their parent's ID. Moving something in the subnav, or just changing the level of one of the subnav nodes will apply this fix
